src/healthimaging-mcp-server/awslabs/healthimaging_mcp_server/server.py

```diff
@@ -0,0 +1,824 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""AWS HealthImaging MCP Server implementation."""
+
+# Standard library imports
+import json
+import logging
```

Comment:
We have standardized on loguru

```suggestion
from loguru import logger
```